src/main/java/io/confluent/copycat/hdfs/RecordWriter.java

```diff
@@ -21,4 +21,5 @@
 public interface RecordWriter<K, V> {
   void write(K key, V value) throws IOException;
   void close() throws IOException;
+  void flush() throws IOException;
```

Comment:
It doesn't seem this is ever invoked? But at least one implementation of `RecordWriter` (in `AvroRecordWriterProvider`) includes code in the method.